### PR TITLE
simple 404 page

### PR DESCRIPTION
**Pull request by sameerav**
screen shot 2015-05-04 at 2 36 21 pm
super simple 404 page

### CODE REVIEW

File: revolv/base/views.py [hunk 1]
```diff
@@ -205,6 +205,8 @@ def get(self, request, *args, **kwargs):
         messages.success(self.request, 'Logged out successfully')
         return redirect('home')
 
+# class PageNotFoundView(View):
```
Comment (noahsark769) [inline revolv/base/views.py]: Can delete?

File: revolv/urls.py [hunk 2]
```diff
@@ -25,3 +25,5 @@
 
     url(r'^', include('cms.urls')),
 )
+
+#handler404 = 'base_views.PageNotFoundView'
```
Comment (noahsark769) [inline revolv/urls.py]: Delete?

File: revolv/templates/404.html [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+{% block head %}
+<link rel="stylesheet" type="text/css" href="{% static "404.css" %}">
+<link href='http://fonts.googleapis.com/css?family=Lato:300,500,800' rel='stylesheet' type='text/css'>
+{% endblock %}
+
+{% block body %}
+<div class="row container">
+	<h1 class="title">We're sorry! This page doesn't exist.</h1>
+	<p class="body-text">Shoot us an email at info@revolv.org to let us know if there's been an error,
```
Comment (noahsark769) [inline revolv/templates/404.html]: It's actually info@re-volv.org, not info@revolv.org

**Comment by noahsark769** [comment]
Also, is this mobilified? If so can i get a screenshot for [tablet | phone] on [portrait | landscape] orientation?

**Comment by sameerav** [comment]
screen shot 2015-05-04 at 2 36 21 pm
screen shot 2015-05-04 at 3 01 37 pm
screen shot 2015-05-04 at 3 02 25 pm
screen shot 2015-05-04 at 3 02 33 pm
screen shot 2015-05-04 at 3 02 42 pm
screen shot 2015-05-04 at 3 02 52 pm

**Comment by noahsark769** [comment]
Reference: fixes #252

**Comment by noahsark769** [comment]
SHIP IT

```
                             |\
                               ||.
                               ||;`
                             ,'|;  :
                           ,': |;  `
                         ,'  | ;  `-`
                       ,'    | ;     :
                     ,'    `-|;      `
                    ;        ;        `
                  ,'      `--\`-.   `--:
            ,-._,'`.         |\||`-.   `
            `;-.`-._`.       | |\ \ `-. `
             :`--`-.`-\  `--.| /`\;\   `-:
             ;      ``-`.    |/  ;  \    `.
             ;           `.`-/   ;   \    `
             ;      :      `/    ;    \    `
     ,-._    |             /     ;   -`\    :
     `;-.`-._| :          /      |      \   `
      ;   `-.|     :     /       |       \   `
      ;      | :    :   /        |        \   :
      ;      | :       /   ;     |         \  `
      ;      |        /   ;      |       --`\  `
      |      |  :    /    ;      |           \  :
      | :    |  :   /            ;            \ `
      |      :     ;            ;:'-.._      -`\ `
      |  :   ;___  ;'---....___;__:__| `-._     \ :
      |      :   `;``-------'';||  :  \    `-._  \`
     -=======:===;===========;=||   :  \       `-.\:
         ___ ; _;_||        ;  ||___,:-.\___,....__\_
        |`--:.|;_o||-______;,..-----""""" __|__...-''
----....|___     """"                 jrei__||-------........________
            """"----....____      __..--''  || ~~~~~
     ~~~ ~~~                """"----....____|/    ~~~~~~~     ~~~~~~~
~~~                ~~~~   ~~~~~~  ~  ~~~       ~~~~
  ~~~~~      ~~~      ~     ~~~~~~~~~ ~~~                ~~~~~~~~~~
```

**Pull request merged**